cmd/sealer/cmd/image/rmi.go

```diff
@@ -22,15 +22,27 @@ import (
 
 var removeOpts *options.RemoveImageOptions
 
+var exampleForRmiCmd = `sealer rmi {imageName or imageID} ...
+
+remove all images:
+        sealer rmi --all/-a
+
+prune dangling images:
+        sealer rmi --prune/-p
+
+force removal of the image and any containers using the image:
+        sealer rmi {imageName or imageID} --force/-f
+
+`
+
 // NewRmiCmd rmiCmd represents the rmi command
 func NewRmiCmd() *cobra.Command {
 	rmiCmd := &cobra.Command{
 		Use:   "rmi",
-		Short: "remove local images by name",
+		Short: "remove local images",
 		// TODO: add long description.
 		Long:    "",
-		Example: `sealer rmi registry.cn-qingdao.aliyuncs.com/sealer-io/kubernetes:v1.19.8`,
-		Args:    cobra.MinimumNArgs(1),
+		Example: exampleForRmiCmd,
```

Comment:
Why to remove `Args: cobra.MinimumNArgs(1)` ?

Reply:
sealer rmi design to remove images in there ways:
* remove specific images (need NArgs)
* remove all images (only need flag)
* remove dangling images (only need flag)

this means no need to specific args num.

Reply:
@tgfree7 Thanks for your detailed explanation. I strongly object to the flag `--all/-a`. This flag is a disaster for sealer. You know every command may be executed casually. If someone executes `sealer rmi -a` carelessly, the world explodes. So we could not provide such an easy way. I learnt some from Docker, and docker does not the `--all` flag in `docker rmi`.  https://docs.docker.com/engine/reference/commandline/rmi/#options . If someone needs to remove all images, please guide him to use complicated command like `for name in `sealer image`; do sealer rmi $name; done`

Could you add a pull request to remove the `--all` flag? @tgfree7

Reply:
Yes, a simple cmd to remove all images is dangerous for a new user.
You mean open a pr to remove the `--all` flag and close this one?

Reply:
`open a pr to remove the --all flag ` is more important. And we can do both in parrallel.

Reply:
OK